02/app.js

```diff
@@ -1,7 +1,29 @@
-
 /* rozwiązanie z pętlą for */
-const x = 4;
+/*const x = prompt("Podaj liczbę!");
+let result = 1;
+
+if (x >= 1 && x <= 9) {
+  for (let i = 1; i <= 9; i++) {
+    result = x * i;
+
+    console.log(x, "*", i, "=", result);
+  }
+} else alert("podaj liczbę z przedziału 1-9");
+ */
+
+/* rozwiązanie z pętlą while  */
 
+const a = prompt("Podaj liczbę podstawy!");
+const n = prompt("Podaj liczbę wykładnika!");
+let number = 1;
 
+while (number <= n) {
+  number++;
 
-/* rozwiązanie z pętlą while  */
\ No newline at end of file
+  if (a > 1 && a < 9 && n > 1 && n < 9) {
+    let result = Math.pow(a, n);
+    console.log(result);
```

Comment:
Zwróć uwagę, żę trzeba to pokazać w konsoli w ten sposób: `3 * 3 * 3 = 27` - spróbuj poprawić.
Podpowiedź: Trzeba użyć pętli.